pstress/generate_mysqld_options.sh

```diff
@@ -0,0 +1,511 @@
+    echo "${OPTION}=DATABASE" >> ${OUTPUT_FILE}
+    echo "${OPTION}=LOGICAL_CLOCK" >> ${OUTPUT_FILE}
+  elif [ "${OPTION}" == "--session-track-system-variables" ]; then
+    echoit "  > Adding possible values time_zone, autocommit, character_set_client, character_set_results, character_set_connection for option '${OPTION}' to the final list..."
+    echo "${OPTION}=time_zone" >> ${OUTPUT_FILE}
+    echo "${OPTION}=autocommit" >> ${OUTPUT_FILE}
+    echo "${OPTION}=character_set_client" >> ${OUTPUT_FILE}
+    echo "${OPTION}=character_set_results" >> ${OUTPUT_FILE}
+    echo "${OPTION}=character_set_connection" >> ${OUTPUT_FILE}
+  elif [ "${OPTION}" == "--tls-version" ]; then
+    echoit "  > Adding possible value TLSv1, TLSv1.1, TLSv1.2, TLSv1.3 for option '${OPTION}' to the final list..."
+    echo "${OPTION}=TLSv1" >> ${OUTPUT_FILE}
+    echo "${OPTION}=TLSv1.1" >> ${OUTPUT_FILE}
+    echo "${OPTION}=TLSv1.2" >> ${OUTPUT_FILE}
+    echo "${OPTION}=TLSv1.3" >> ${OUTPUT_FILE}
+  elif [ "${OPTION}" == "--upgrade" ]; then
+    echoit "  > Adding possible value NONE, MINIMAL, AUTO, FORCE for option '${OPTION}' to the final list..."
+    echo "${OPTION}=NONE" >> ${OUTPUT_FILE}
+    echo "${OPTION}=MINIMAL" >> ${OUTPUT_FILE}
+    echo "${OPTION}=AUTO" >> ${OUTPUT_FILE}
+    echo "${OPTION}=FORCE" >> ${OUTPUT_FILE}
+  elif [ "${OPTION}" == "--binlog-checksum" ]; then
+    echoit "  > Adding possible value NONE, CRC32 for option '${OPTION}' to the final list..."
+    echo "${OPTION}=NONE" >> ${OUTPUT_FILE}
+    echo "${OPTION}=CRC32" >> ${OUTPUT_FILE}
+  elif [ "${OPTION}" == "--event-scheduler" ]; then
+    echoit "  > Adding possible value ON, OFF, DISABLED for option '${OPTION}' to the final list..."
+    echo "${OPTION}=ON" >> ${OUTPUT_FILE}
+    echo "${OPTION}=OFF" >> ${OUTPUT_FILE}
+    echo "${OPTION}=DISABLED" >> ${OUTPUT_FILE}
+  elif [ "${OPTION}" == "--gtid-mode" ]; then
+    echoit "  > Adding possible value ON, OFF, OFF_PERMISSIVE, ON_PERMISSIVE for option '${OPTION}' to the final list..."
+    echo "${OPTION}=ON --enforce-gtid-consistency=ON " >> ${OUTPUT_FILE}
+    echo "${OPTION}=OFF" >> ${OUTPUT_FILE}
+    echo "${OPTION}=OFF_PERMISSIVE" >> ${OUTPUT_FILE}
+    echo "${OPTION}=ON_PERMISSIVE" >> ${OUTPUT_FILE}  
+  elif [ "${VALUE}" == "TRUE" -o "${VALUE}" == "FALSE" -o "${VALUE}" == "ON" -o "${VALUE}" == "OFF" -o "${VALUE}" == "YES" -o "${VALUE}" == "NO" ]; then
+    echoit "  > Adding possible values TRUE/ON/YES/1 and FALSE/OFF/NO/0 (as a universal 1 and 0) for option '${OPTION}' to the final list..."
+    echo "${OPTION}=1" >> ${OUTPUT_FILE}
+    echo "${OPTION}=0" >> ${OUTPUT_FILE}
+  elif [[ "$(echo ${VALUE} | tr -d ' ')" =~ ^-?[0-9]+$ ]]; then
+    if [ "${VALUE}" != "0" ]; then 
+    echoit "  > Adding int values (${VALUE}, -1, 0, 1, 2, 12, 24, 254, 1023, 2047, -1125899906842624, 1125899906842624) for option '${OPTION}' to the final list..."
+    echo "${OPTION}=${VALUE}" >> ${OUTPUT_FILE}
+  else
```

Comment:
Indentation is wrong, else is part of inner if block

Reply:
fixed